src/plugins/intel_gpu/tests/unit/test_cases/crop_gpu_test.cpp

```diff
@@ -1576,3 +1576,33 @@ TEST(crop_gpu, optimized_out_crop) {
     ASSERT_TRUE(all_primitives["crop1"] == "_optimized_");
     ASSERT_TRUE(all_primitives["crop2"] == "_optimized_");
 }
+
+TEST(crop_single_axis, simple_Baxis) {
+    auto& engine = get_test_engine();
+
+    auto input1 = engine.allocate_memory({ data_types::f32, format::bfyx, tensor{ 3, 2, 1, 2 } });
+
+    set_values(input1, {
+        1.f, 2.f,  3.f,  4.f,
+        5.f, 6.f,  7.f,  8.f,
+        9.f, 10.f, 11.f, 12.f
+    });
+
+    topology topology;
+    topology.add(input_layout("Input", input1->get_layout()));
+    topology.add(crop("crop", input_info("Input"), tensor{1, 2, 1, 2}, tensor(1, 0, 0, 0)));
+    topology.add(reorder("reorder", input_info("crop"), format::bfyx, data_types::i8));
+
+    network network(engine, topology, get_test_default_config(engine));
+
+    network.set_input_data("Input", input1);
+
+    auto outputs = network.execute();
+
+    auto output = outputs.at("reorder").get_memory();
+    cldnn::mem_lock<int8_t> output_ptr(output, get_test_stream());
+
+    std::vector<int8_t> expected_results = {
+        5, 6, 7, 8
+    };
```

Comment:
could you add crop test for opt-out?

Reply:
This test also pass through the logic.
[ RUN      ] crop_single_axis.simple_Baxis
    -- prepare_buffer_fusing::run
    -- can_crop_be_optimized_along_batch()

Reply:
Added test into 'prepare_buffer_fusing'